ISHPullUp/ISHPullUpViewController.m

```diff
@@ -494,12 +494,17 @@ - (BOOL)dimmingViewShouldBeHidden {
 
     CGFloat maximumHeightOverMinimum = self.maximumBottomHeightCached - self.minimumBottomHeightCached;
 
-    if (!maximumHeightOverMinimum) {
+    if (maximumHeightOverMinimum <= 0) {
         // if the view cannot be extended beyond minimum always hide dimming view
         return YES;
     }
 
     CGFloat heightOverMinimum = self.bottomHeight - self.minimumBottomHeightCached;
+    if (heightOverMinimum <= 0) {
```

Comment:
Is this check also necessary? It seems that it would be caught by our normal return expression?